### ISSUE TITLE
No data in FMC after the last MSFS update.

### ISSUE THREAD

**Issue by dudsha** [Heavy-Division/B78XH#216]
No data in FMC after the last MSFS update and nothing is working :-(

**Comment by ace2029** [comment]
Can confirm. I had numerous issues after updating to SU9 and using the SU9 build for the 787. CTD's/freezes, FMC would not accept any airports/route import from Simbrief. Any other aircraft was fine, no crashes at all.

**Comment by phnkone** [comment]
start the 787 asobo first, and then reinstall the mod. This is how it worked for me

**Comment by almaxflight** [comment]
Hi, I also confirm that the 787 fmc does not accept any data, route, icao airport and airways. This after the last update 1.25.7... tank you

**Comment by simondaysh** [comment]
Hi, I'm having this issue, too.

**Comment by isobar5** [comment]
Can also confirm Sim update 9 broke it.

**Comment by ColeHardTruth** [comment]
Ditto; I don't know how I get so unlucky, I finally pulled the trigger on the Premium Deluxe upgrade and the Heavy mod isn't working! 😭

**Comment by simondaysh** [comment]
> Ok... But someone try to install a precedent version? Maybe the 0.12 works.. I Hope thath the developers to fix this ASAP! :( :(:(

I have just tried 0.12 - and I still can't enter a flightplan :(

**Comment by almaxflight** [comment]
Great!! As sono as possibile i will try... Thank's

**Comment by ColeHardTruth** [comment]
> 

Just tried it, now none of the displays (MFD, EICAS, FMC) work...

**Comment by por930** [comment]
The file mentioned by simondaysh works great and the FMC is functional...so far. I can now get from WSSS to RJAA

**Comment by almaxflight** [comment]
It happened to me too, even in previous versions. it happened with the flight plans imported from simbrief. from that moment on I always set FMC manually and no problem :)

**Comment by por930** [comment]
Ummm, interesting. This is the first time I have had any issues with the MCDU refusing to both load the Simbrief Flight plan, and then afterwards, the FMC departure and Destination identifiers were not possible. Using the File supplied above resolved the MCDU entry. I will try a few manual entries and see how they operate.

**Comment by almaxflight** [comment]
> Hi - I've just been told by KimLada to download the main file - https://github.com/Heavy-Division/B78XH/archive/refs/heads/main.zip - I've done this, and it's loaded the flight plan! :) Enjoy!


Yes! It works! 
But in ND not appear T/C and TOD.. :(

**Comment by por930** [comment]
When I saved the flight yesterday, and tried to resume today, the load did not work. No flight plan loaded. I need to do some testing of the save/load feature for various aircraft to see if this is an issue only with the 787 heavy division.

**Comment by NakajimaYoshi** [comment]
Hi all, 
Please make sure when updating the mod that you fully delete the current version in your community folder, and aren't just overwriting it with the updated file. We will be moving towards a more standardized bug reporting system on the github, so if you plan on posting issues here please FIRST try recreating the problem on the Asobo DEFAULT 787 and submitting a bug reporting using a template, or it will not be read. For support, please use the discord. Thanks

**Comment by por930** [comment]
Thanks for the comments. Look forward to the 787 improving.
When I installed the update file above, I first removed the existing mod from the community folder, then installed new file. This immediately allowed the flight plan to be correctly loaded. I had a missed waypoint issue. came off AP, manually put the 787 back on track, then set a DIRECT TO waypoint, from then, all ok. The save /reload was terrible.
So, are you saying, any issues with the Heavy Division mod, report here. Any issues with the default 787, of course, report in the the Asobo discord.

**Comment by NakajimaYoshi** [comment]
That is correct. As the 787 heavy is not a standalone plane, but an enhancement of the default, it's up to the user to first discern whether the issue is related to the base plane from Asobo first before submitting bug reports. This greatly reduces the workload for us and makes it easier to track/solve issues related to the mod, as we don't have access to many core files of the plane that were written by Asobo
